nodes/Name.py

```diff
@@ -0,0 +1 @@
+[l+l for l in 'str']
```

Comment:
This is interesting. Is not just a single name like `my_var` enough to make the `Name` node?